libs/utils/tsconfig.build.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["src", "src/data/*.json"],
+  "include": ["src", "src/data/*.json", "src/data/*.zip"],
```

Comment:
These changes were needed to get libs/utils/src/data/ballot-package-state-of-hamilton.zip to build. Tests were failing because that zip file had no `systemSettings.json` file. Since I later introduced a fallback to load `DEFAULT_SYSTEM_SETTINGS`, the update to `ballot-package-state-of-hamilton.zip` isn't strictly necessary, but is good hygiene to keep.